RetailSuiteAccounting/UserInterface/01a_Bookings.md

```diff
@@ -0,0 +1,424 @@
+
+- ![Left arrow](../../Assets/Icons/Back02.png "[Left arrow]") (Left arrow)   
+Click this arrow to open the *Post extras* side bar. For detailed information about searching in the *Accounting* module, see [Search in accounting](../Operation/14_SearchAccounting.md).
+
+- *Show (all/the last [XXX]) postings*    
+Click the drop-down list to select the preferred display filter.
+
+
+- *Amount*  
+The total amount of all postings displayed.
+
+  - *Amount 1*  
+  Click this button to manually set an amount 1. A new window opens where the amount can be entered and confirmed with [OK].
+  - *Amount 2*  
+  Click this button to manually set an amount 2. A new window opens.  A new window opens where the amount can be entered and confirmed with [OK].
+
+[comment]: <> (Function of Summe 1 & Summe 2??? Describe new window? It seems to change automatically sometimes, at least Summe 1, but I could not find out why! RS FH!)
+
+
+The input line allows to create a posting manually. For detailed information, see [Create a manual posting](../Operation/04_CreateManualBooking.md).
+
+
+- *Seq. no.*  
+ Enter the posting number. Posting numbers must be sequential. The system adds the next available number automatically if the field is left empty.
+
+- *Currency code*  
+Click the drop-down menu to select the appropriate currency.  
+
+  > [Info] It is possible to set up accounts and post transactions in different currencies, but the *Accounting* module can only work with the base currency configured in the system. For detailed information about managing and posting exchange rate fluctuations, see [Post exchange rate fluctuations](../Operation/12_BookExchangeRateFluctuations.md).
+
+- *Exchange rate*  
+This field updates automatically when changing the currency. It can also be modified manually.
+
+  > [Info] The system retrieves once a day the current foreign currency exchange rates as published by the ECB and applies them automatically, therefore always working with the most up-to-date information.
+
+- *Turnover*  
+Enter the turnover value. Turnover values can be positive (incomes) or negative (expenses). This field is compulsory.
+
+- *Contra account*  
+Enter the appropriate contra account. This field is compulsory.
+
+  > [Info] After you have entered at least two numbers in the field, a list is displayed below the field with all accounts matching the entry. You can also select the corresponding account in the list. Alternatively, use the search function to find the account number, see [Search for an account](../Operation/13_SearchAccounting.md#search-for-an-account).
+
+
+- *Receipt1*  
+Enter an appropriate reference value, such as the invoice number.
+
+  > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
```

Comment:
```suggestion
  > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
```
```suggestion
  > [Info] Even though it is not mandatory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent postings taking this field as reference, for example for an invoice settlement or a credit note in case of refund.
```